search/search-tool/tool/src/java/org/sakaiproject/search/tool/ControllerServlet2.java

```diff
@@ -187,6 +188,17 @@ public void execute(HttpServletRequest request, HttpServletResponse response)
 		String skin = "default/"; // this could be changed in the future to
 		// make search skin awaire
 
+		Locale locale = (new ResourceLoader()).getLocale();
```

Comment:
I think you can use Locale.toLanguageTag() which should do all this for you.

http://www.rfc-editor.org/rfc/bcp/bcp47.txt